src/pages/home/report/OnboardingReportFooterMessage.tsx

```diff
@@ -0,0 +1,107 @@
+
+type OnboardingReportFooterMessageOnyxProps = {
+    /** Saved onboarding purpose selected by the user */
+    choice: OnyxEntry<OnboardingPurposeType>;
+
+    /** Collection of reports */
+    reports: OnyxCollection<Report>;
+
+    /** The list of this user's policies */
+    policies: OnyxCollection<PolicyType>;
+};
+type OnboardingReportFooterMessageProps = OnboardingReportFooterMessageOnyxProps;
+
+function OnboardingReportFooterMessage({choice, reports, policies}: OnboardingReportFooterMessageProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const {isSmallScreenWidth} = useWindowDimensions();
+
+    const adminChatReport = useMemo(() => {
+        const adminsReports = Object.values(reports ?? {}).filter((report) => report?.chatType === CONST.REPORT.CHAT_TYPE.POLICY_ADMINS);
+        const activePolicies = Object.values(policies ?? {}).filter((policy) => PolicyUtils.shouldShowPolicy(policy, false));
+
+        return adminsReports.find((report) => activePolicies.find((policy) => policy?.id === report?.policyID));
+    }, [policies, reports]);
+
+    const content = useMemo(() => {
+        switch (choice) {
+            case CONST.ONBOARDING_CHOICES.MANAGE_TEAM:
+                return (
+                    <>
+                        {translate('onboardingBottomMessage.newDotManageTeam.phrase1')}
+                        <TextLink
+                            style={styles.label}
+                            onPress={() => Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(adminChatReport?.reportID ?? ''))}
+                        >
+                            {adminChatReport?.reportName ?? CONST.REPORT.WORKSPACE_CHAT_ROOMS.ADMINS}
+                        </TextLink>
+                        {translate('onboardingBottomMessage.newDotManageTeam.phrase2')}
+                    </>
+                );
+            default:
+                return (
+                    <>
+                        {translate('onboardingBottomMessage.default.phrase1')}
+                        <TextLink
+                            style={styles.label}
+                            onPress={() => ReportInstance.navigateToConciergeChat()}
+                        >
+                            {`${CONST?.CONCIERGE_CHAT_NAME}`}
+                        </TextLink>
+                        {translate('onboardingBottomMessage.default.phrase2')}
+                    </>
+                );
+        }
+    }, [adminChatReport?.reportName, adminChatReport?.reportID, choice, styles.label, translate]);
+
+    return (
+        <View
+            style={[
+                styles.chatFooter,
+                isSmallScreenWidth ? styles.mb5 : styles.mb4,
+                styles.mh5,
+                styles.mt4,
+                styles.flexRow,
+                styles.alignItemsCenter,
+                styles.p4,
+                styles.borderRadiusComponentLarge,
+                styles.hoveredComponentBG,
```

Comment:
Why hoveredComponentBG and not just a regular componentBG background color? What did we have in the mocks? cc @Expensify/design

Reply:
I updated the component to reuse the [Banner](https://github.com/Expensify/App/pull/40010#issuecomment-2077184665) under the hood - https://github.com/Expensify/App/pull/40010#issuecomment-2077184665.